src/android/com/microsoftopentechnologies/intellij/wizards/createvm/CreateStorageAccountForm.java

```diff
@@ -0,0 +1,250 @@
+/**
+ * Copyright 2014 Microsoft Open Technologies Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.microsoftopentechnologies.intellij.wizards.createvm;
+
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.progress.ProgressIndicator;
+import com.intellij.openapi.progress.ProgressManager;
+import com.intellij.openapi.progress.Task;
+import com.intellij.openapi.project.Project;
+import com.microsoftopentechnologies.intellij.helpers.UIHelper;
+import com.microsoftopentechnologies.intellij.helpers.azure.AzureCmdException;
+import com.microsoftopentechnologies.intellij.helpers.azure.sdk.AzureSDKManagerImpl;
+import com.microsoftopentechnologies.intellij.model.Subscription;
+import com.microsoftopentechnologies.intellij.model.vm.AffinityGroup;
+import com.microsoftopentechnologies.intellij.model.vm.CloudService;
+import com.microsoftopentechnologies.intellij.model.vm.Location;
+import com.microsoftopentechnologies.intellij.model.vm.StorageAccount;
+
+import javax.swing.*;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
+import java.awt.*;
+import java.awt.List;
+import java.awt.event.*;
+import java.util.*;
+
+
+public class CreateStorageAccountForm extends JDialog {
+    private JPanel contentPane;
+    private JButton buttonOK;
+    private JButton buttonCancel;
+    private JComboBox subscriptionComboBox;
+    private JTextField nameTextField;
+    private JComboBox regionOrAffinityGroupComboBox;
+    private JComboBox replicationComboBox;
+    private JProgressBar createProgressBar;
+
+    private Runnable onCreate;
+    private Subscription subscription;
+    private StorageAccount storageAccount;
+
+    private enum ReplicationTypes {
+        Standard_LRS,
+        Standard_GRS,
+        Standard_RAGRS;
+
+        public String getDescription() {
+            switch (this) {
+                case Standard_GRS:
+                    return "Geo-Redundant";
+                case Standard_LRS:
+                    return "Locally Redundant";
+                case Standard_RAGRS:
+                    return "Read_Access Geo-Redundant";
```

Comment:
This is text displayed to the user? If yes, we should remove the '_' for `Standard_RAGRS`.